lib/tasks/envirotech.rake

```diff
@@ -0,0 +1,7 @@
+namespace :envirotech do
+  desc 'Update issue_solution_regulation.json file with latest data from toolkit scraper'
+  task update_local_relational_data: :environment do
+    $stdout.reopen("data/envirotech/issue_solution_regulation.json", "w")
+    puts JSON.pretty_generate(Envirotech::ToolkitScraper.new.all_issue_info)
```

Comment:
This seems like a rather exotic way to [write to a file](http://alvinalexander.com/blog/post/ruby/how-write-text-to-file-ruby-example) :smiley: